src/app/auth/components/AuthButtons.tsx

```diff
@@ -4,10 +4,13 @@ import { Lock, User } from 'lucide-react';
 const AuthButtons: React.FC = () => {
   return (
     <div className="space-y-6">
-      <button className="w-full flex items-center justify-center px-6 py-3 bg-blue-600 text-white rounded-xl hover:bg-blue-700 transition-all shadow-md group focus:ring-2 focus:ring-blue-300 focus:outline-none">
+      <a
```

Comment:
Can we use `Link` component instead of `<a>` because I believe the `Link` component helps with pre-fetching and caching as I see in the documentation. 

https://nextjs.org/docs/app/building-your-application/routing/linking-and-navigating